back/engines/commercial/id_vienna_saml/config/initializers/omniauth.rb

```diff
@@ -15,7 +15,5 @@
 end
 
 Rails.application.config.middleware.use OmniAuth::Builder do
-  options = { setup: VIENNA_SAML_CITIZEN_SAML_SETUP_PROC, name: 'vienna_citizen' }
-  options[:sp_entity_id] = 'citizenlabtest' if Rails.env.development? || Rails.env.test?
```

Comment:
Some time ago, I added this line because I didn't know it can be configured in `ENVIRONMENTS`. But anyway, I have not managed to make it work locally.